cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,78 @@
+import { faker } from '@faker-js/faker';
+import PageObject from '../PageObject';
+
+class SettingsPageObject extends PageObject {
+  url = '/#/settings';
+
+  get bioField() {
+    return cy.findByPlaceholder('Short bio about you');
+  }
+
+  get emailField() {
+    return cy.findByPlaceholder('Email');
+  }
+
+  get userNameField() {
+    return cy.findByPlaceholder('Your username');
+  }
+
+  get passwordField() {
+    return cy.findByPlaceholder('Password');
+  }
+
+  get updateSettingsBtn() {
+    return cy.get('.btn.btn-lg');
+  }
+
+  checkUpdate() {
+    cy.get('.swal-modal').should('contain', 'Update successful!');
+  }
```

Comment:
The 'checkUpdate' method checks for a specific success message ('Update successful!'). If this message changes in the application, the test will fail. Consider making this check more flexible or configurable to avoid brittle tests.